golang/dashboard/README.md

```diff
@@ -38,12 +38,27 @@ $ echo "export PATH=\$PATH:\$GOBIN" >> ~/.bash_profile
 $ source ~/.bash_profile
 ```
 
+::: tip Troubleshooting
+If, after following steps 3 and 4 below, you are not able to successfully run `make install` or `make init` then try replacing the above lines with the following:
+
+```
+export GOPATH=$HOME/go
+export PATH=$GOPATH/bin:$PATH
+```
+
+Don't forget to run:
+```
+source ~/.bash_profile
+```
+
+You may even need to restart your terminal.
+:::
+
```

Comment:
Just some formatting suggestions:
```
 > *Troubleshooting tip*
 >
 > If, after following steps 3 and 4 below, you are not able to successfully run `make install` or `make init` then try replacing the above lines with the following:
 >
 >```sh
 > export GOPATH=$HOME/go
 > export PATH=$GOPATH/bin:$PATH
 > ```
 >
 > Don't forget to run:
 > ```sh
 > source ~/.bash_profile
 > ```
 >
 > You may even need to restart your terminal.
```

 - Turns it into a block quote by adding `>` before each line
 - Adds "sh" to code blocks for syntax highlighting
 - Italicizes "Troubleshooting tips" by adding `*`s

Reply:
Does `tip` do the same thing as block quote?

Reply:
I'm not sure what `tip` does.  It looks like it wasn't working though 
<img width="939" alt="Screen Shot 2020-04-30 at 3 11 29 PM" src="https://user-images.githubusercontent.com/36751902/80759871-393ad900-8af5-11ea-933c-12b9983a9f7c.png">

Reply:
oh, maybe it's a vuepress thing. I grabbed it off the agoric documentation. hehe...

Reply:
Yeah, I just figured out it was Vuepress lol